[2024-03-30T22:01:47.021+0000] {processor.py:154} INFO - Started process (PID=2483) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:01:47.022+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:01:47.023+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:01:47.023+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:01:47.034+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:01:47.033+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:01:47.035+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:01:47.045+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.026 seconds
[2024-03-30T22:02:17.518+0000] {processor.py:154} INFO - Started process (PID=2517) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:02:17.519+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:02:17.521+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:02:17.520+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:02:17.538+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:02:17.537+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:02:17.539+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:02:17.554+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.040 seconds
[2024-03-30T22:02:48.023+0000] {processor.py:154} INFO - Started process (PID=2552) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:02:48.024+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:02:48.025+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:02:48.025+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:02:48.042+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:02:48.041+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:02:48.043+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:02:48.056+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.038 seconds
[2024-03-30T22:03:18.496+0000] {processor.py:154} INFO - Started process (PID=2586) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:03:18.498+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:03:18.498+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:03:18.498+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:03:18.518+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:03:18.516+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:03:18.518+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:03:18.533+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.042 seconds
[2024-03-30T22:03:48.955+0000] {processor.py:154} INFO - Started process (PID=2620) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:03:48.957+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:03:48.958+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:03:48.958+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:03:48.977+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:03:48.975+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:03:48.977+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:03:48.991+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.042 seconds
[2024-03-30T22:04:19.420+0000] {processor.py:154} INFO - Started process (PID=2654) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:04:19.421+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:04:19.422+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:04:19.422+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:04:19.442+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:04:19.440+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:04:19.443+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:04:19.456+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.040 seconds
[2024-03-30T22:04:49.942+0000] {processor.py:154} INFO - Started process (PID=2688) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:04:49.943+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:04:49.944+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:04:49.944+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:04:49.962+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:04:49.961+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:04:49.963+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:04:49.978+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.040 seconds
[2024-03-30T22:05:20.409+0000] {processor.py:154} INFO - Started process (PID=2720) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:05:20.409+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:05:20.410+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:05:20.410+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:05:20.419+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:05:20.418+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:05:20.419+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:05:20.428+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.021 seconds
[2024-03-30T22:05:50.920+0000] {processor.py:154} INFO - Started process (PID=2747) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:05:50.922+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:05:50.923+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:05:50.923+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:05:50.944+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:05:50.943+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:05:50.945+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:05:50.959+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.045 seconds
[2024-03-30T22:06:21.402+0000] {processor.py:154} INFO - Started process (PID=2781) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:06:21.403+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:06:21.404+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:06:21.404+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:06:21.424+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:06:21.423+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:06:21.424+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:06:21.438+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.040 seconds
[2024-03-30T22:06:51.859+0000] {processor.py:154} INFO - Started process (PID=2815) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:06:51.860+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:06:51.861+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:06:51.861+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:06:51.878+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:06:51.877+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/subdags/subdag_downlads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/subdags/subdag_downlads.py", line 4, in <module>
    from subdags.subdag_downloads import subdag_downloads
ModuleNotFoundError: No module named 'subdags.subdag_downloads'
[2024-03-30T22:06:51.879+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:06:51.891+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.036 seconds
[2024-03-30T22:07:06.183+0000] {processor.py:154} INFO - Started process (PID=2830) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:07:06.185+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:07:06.186+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:07:06.186+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:07:06.195+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:07:06.210+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.033 seconds
[2024-03-30T22:07:36.725+0000] {processor.py:154} INFO - Started process (PID=2865) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:07:36.726+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:07:36.726+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:07:36.726+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:07:36.731+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:07:36.742+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.020 seconds
[2024-03-30T22:08:07.256+0000] {processor.py:154} INFO - Started process (PID=2899) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:08:07.258+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:08:07.259+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:08:07.259+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:08:07.266+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:08:07.281+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.030 seconds
[2024-03-30T22:08:37.762+0000] {processor.py:154} INFO - Started process (PID=2932) to work on /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:08:37.763+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/subdags/subdag_downlads.py for tasks to queue
[2024-03-30T22:08:37.764+0000] {logging_mixin.py:120} INFO - [2024-03-30T22:08:37.764+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:08:37.771+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/subdags/subdag_downlads.py
[2024-03-30T22:08:37.784+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/subdags/subdag_downlads.py took 0.026 seconds
